pages/index.tsx

```diff
@@ -1,3 +1,18 @@
+import Head from 'next/head';
+
 export default function Home() {
-  return null;
+  return (
+    <div className="gel-wrap home-page">
+      <Head>
+        <title>Roseate Cards</title>
+      </Head>
+
+      <a className="home__featured" href="/cards/christmas">
```

Comment:
Need to wrap this in a `<Link />` so that client-side routing works.
e.g. 
```
<Link href="/cards/christmas">
  <a className="home__featured">
    Stuff
  </a>
</Link>
```